tests/mapped_struct.py

```diff
@@ -1007,9 +1014,9 @@ def _testBuild(self, N, tempdir, **gen_kwargs):
             elem = rv.get(k)
             if isinstance(elem, list):
                 for e in elem:
-                    self.assertTrue(isinstance(e, (int, long)))
+                    self.assertTrue(isinstance(e, (int, long, numpy.uint32)))
```

Comment:
No, we can't allow numpy types here.

If this is for pure python tests, I'd say rollback. We can require Cython.

Reply:
why not numpy types?